### DISCUSSION TITLE
Stop contacting *.docker.io ?

### DISCUSSION THREAD

**Discussion by unbranched** [containrrr/watchtower#1651] [Q&A] [answered]
Hi, I noticed that watchtower keeps contacting the domains "index.docker.io" and "auth.docker.io" every 2-3 minutes, I guess when it check for updates.
I need to limit the requests in my netowork dns, so I'd like to stop this behaviour.
Considering that I need to check for image updates only from my local repo, is there a way to disable the check of the official docker hub?

Thanks

**Comment by piksel** [comment, marked as answer]
As long as you only have containers with images on your private repository it would only need to access it.

**Reply by unbranched** [reply]
Apologize, maybe I don't understand.
Yes I only have containers with images on my private repository, so can I stop it contacting docker.io ?

**Reply by piksel** [reply]
I assume you are running watchtower from docker.io? If so, you can limit it's scope to just the containers you need by supplying their names on the command line or using enable labels.

**Reply by unbranched** [reply]
Yes. I'm already supplying their names on the command line. I'm not using the enable labels.

**Reply by piksel** [reply]
It should say in the logs what it's updating. It's a bit more clear if you increase the logging level using `--debug`.

**Reply by unbranched** [reply]
Well it was under my eyes... a coworker put another container under checking that originated from docker hub.
Solved, thanks piksel.
